mobify/generator-nightwatch
# Add coverage.json

CSOPS-118: Add inventory.json to generator-nightwatch

Status: **Opened for visibility**
Reviewers: *\* @scalvert @marlowpayne **
JIRA: [CSOPS-118](https://mobify.atlassian.net/browse/CSOPS-118)
## Changes
- Add inventory.json template to generator-nightwatch
- Update index.js to include the copying of inventory.json
- Renamed pages/index.js to home.js and updated strings
## How to test-drive this PR
- `npm install` this branch
- `yo nightwatch`
- ensure inventory.json is generated in the tests folder
- `grunt nightwatch`
- ensure tests run
